packages/core/content-type-builder/server/src/services/builder.ts

```diff
@@ -8,25 +8,38 @@ export const getReservedNames = () => {
       'time',
       'upload',
       'then', // https://github.com/strapi/strapi/issues/15557
-      'rest', // https://github.com/strapi/strapi/issues/13643
```

Comment:
Just a note: this has been fixed in v5 with this PR https://github.com/strapi/strapi/pull/19365